src/utils/AutoTextFit.jsx

```diff
@@ -52,17 +52,36 @@ const DEFAULTS = {
   includeWidth: false,
 }
 
+const childrenElementsWider = (childrenElements, parentWidth) => {
+  return childrenElements.some(element => {
+    if (hasChildren(element)) {
+      return childrenElementsWider([...element.children], parentWidth)
+    }
+    if (isWiderThanParent(element, parentWidth)) {
+      return true
+    }
+    return false
+  })
+}
+
+const hasChildren = element => element.childElementCount > 0
+const isWiderThanParent = (element, parentWidth) => element.offsetWidth > parentWidth
```

Comment:
move _hasChildren_ and _isWiderThanParent_ above the function it is actually used